### ISSUE TITLE
Multiple collapsible sections under one collapsible section

### ISSUE THREAD

**Issue by JaiKul** [Bioruebe/markdown-it-collapsible#1]
I am trying to add 2 collapsible sections inside one collapsible section. Is this possible?
My markdown is something like this
```
+++ Parent collapsible section
Some text
+++ Child 1 collapsible
Child 1 text
+++ Child 2 collapsible
Child 2 text
+++
```

When I run this, I get
```
⮟ Parent collapsible section
    Some text

    ⮟ Child 1 collapsible
        Child 1 text

        ⮟ Child 2 collapsible
            Child 2 text
```

How do I get the following instead?
```
⮟ Parent collapsible section
    Some text

    ⮟ Child 1 collapsible
        Child 1 text

    ⮟ Child 2 collapsible
        Child 2 text
```

If I add `+++` after "Child 1 text", then "Child 2 collapsible" becomes a new collapsible section at the same level as "Parent collapsible section". Adding a blank/new line doesn't have any effect.

**Comment by Bioruebe** [comment]
Yes, it is possible, and you almost got it right.
The only thing you need to do is add more `+` characters to the parent. Like this:
```markdown
++++ Parent collapsible section
  Some text
  
  +++ Child 1 collapsible
  Child 1 text
  +++
  
  +++ Child 2 collapsible
  Child 2 text
  +++
++++
```

It doesn't matter how many, only that the start and end tag have the same number of `+` and child elements have less than the parent. Indention and newlines are not necessary (and probably not allowed when using stricter markdown parsers), I added them just for readability.

![Screenshot](https://user-images.githubusercontent.com/3521951/151049046-9c80ace4-3869-49cd-ad3d-c0f9c250fb2b.png)

**Comment by JaiKul** [comment]
Thanks @Bioruebe !
Before closing the issue, could I also ask if there is a way to keep the section open initially?
I understand, we just need to add the attribute `open` in the `<details>` tag but could this be done through markdown when using this package?

**Comment by Bioruebe** [comment]
No, that's not possible in the current version. Feel free to open another issue, if you want this feature to be added :)

**Comment by Bioruebe** [comment]
I just released version 2.0.0, which allows collapsibles being [open by default](https://github.com/Bioruebe/markdown-it-collapsible#open-state).
